scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -47,6 +49,11 @@ class ThemeAggregator {
   static aggregateTheme(themeName, options = {}, defaultFlag) {
     const { theme, scoped = [] } = options;
 
+    if (!themeName) {
+      Logger.warn(`Failed to aggregate '${themeName}'. Falsey values are not accepted.`);
```

Comment:
If themeName is undefined, this message isn't going to be meaningful since consumers on how to fix this.